src/main/java/homework/day8/Cars.java

```diff
@@ -29,8 +29,9 @@ public static void main(String[] args) throws IOException {
         logs.close();
 
         for (int i = 0; i < cars.size(); i++) {
-            if (cars.get(i).length() > 4) {
+            if ((cars.get(i).length() > 4)) {
```

Comment:
why did you add more parentheses?)

Reply:
It was overcoding to check that if return'true'
I will delete it